CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Use regular components instead of `Label` in `RadioGroupItem` @layershifter ([#XXX](https://github.com/stardust-ui/react/pull/XXX))
```

Comment:
Please add section above this (Breaking change probably if we introduce changes of the typings)